### ISSUE TITLE
ADMIN Game management page

### ISSUE THREAD

**Issue by portaljumper007** [PirateGame/Pirate-Game-Frontend#8]

**Comment by Alex40144** [comment]
can view everything about a game,
can remove questions and events if game gets hung

**Comment by portaljumper007** [comment]
> can view everything about a game,
> can remove questions and events if game gets hung

Yes, though the removal of questions and events would be complicated and I will prioritise making them work first.

**Comment by Alex40144** [comment]
maybe we could just let the admin give a response instead of removing them

**Comment by portaljumper007** [comment]
> maybe we could just let the admin give a response instead of removing them

solid suggestion!
